module/x/gravity/keeper/pool.go

```diff
@@ -153,6 +154,7 @@ func (k Keeper) RemoveFromOutgoingPoolAndRefund(ctx sdk.Context, txId uint64, se
 // WARNING: Do not make this function public
 func (k Keeper) addUnbatchedTX(ctx sdk.Context, val *types.InternalOutgoingTransferTx) error {
 	store := ctx.KVStore(k.storeKey)
+	log.Println("QUA", types.GetOutgoingTxPoolKey(*val.Erc20Fee, val.Id))
```

Comment:
Remove?

Reply:
Whoops! Let me do a cmd+F and remove all debug prints